File: find-the-mine/findTheMine.js
const mineLocation = (array) => {
    const output = [];

    array.forEach((element, index) => {
        element.forEach((innerElement, innerIndex) => {
            if(innerElement === 1){
                output.push(index, innerIndex);
            };
        });
    });
    
    return output;
};

module.exports = { mineLocation };

//https://www.codewars.com/kata/528d9adf0e03778b9e00067e/train/javascript

File: python-venv/find_the_mine/kata_function/find_the_mine.py
def mine_location(array):
    for i in range (len(array)):
        for j in range(len(array[i])):
            if array[i][j] == 1:
                return [i, j]
            
#https://www.codewars.com/kata/528d9adf0e03778b9e00067e/train/python